src/Hng.Web/Controllers/LanguageController.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Hng.Application.Features.Languages.Dtos;
+using Hng.Application.Features.Languages.Queries;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Hng.Web.Controllers
+{
+    [Route("api/[controller]")]
```

Comment:
Please confirm if controller name matches specification in nestjs docs

Reply:
Yes, it matches nestjs docs specification